lib/tasks/update_vendors.rake

```diff
@@ -0,0 +1,20 @@
+desc 'Update provider/vendor associations from yml file'
+task update_vendors: %i[environment] do
```

Comment:
do we want to add this to the local dev setup (to run in Heroku) and in normal deploys or later?

Reply:
Maybe a persistent automatic data migration instead? This would also ensure the task runs at least once on all environments and would allow me to write some tests, too.

Reply:
spoke in person: not adding to deploy now, just dev setup and a data migration to populate

Reply:
We have moved the rake task into a service, added tests and we are now calling this service from both the rake task and the one-off `AddVendorsToProvidersForFirstTime` data migration (for real environments). The rake task is also invoked during `setup_local_dev_data`, which should take care of review apps and local dev.